_data/projects/Encryptor.yml

```diff
@@ -0,0 +1,13 @@
+name: Encryptor
+desc: A password encryptor package for Linux
+site: https://github.com/ScorchingShade/Encryptor
+tags:
+- python
+- encryption
+- aes
+- up-for-grabs
```

Comment:
Can you remove this tag and the two below? These represent details about your project, and the label used is covered in the next section...

Reply:
Sorry, I'll keep in mind from now on!